apps/etterlatte-saksbehandling-ui/client/src/components/person/AktivitetspliktInfoModal.tsx

```diff
@@ -332,15 +334,17 @@ export const AktivitetspliktInfoModal = ({
                   Den etterlatte skal informeres om aktivitetskravet som vil tre i kraft 6 måneder etter dødsfallet. Det
                   skal opprettes et manuelt informasjonsbrev som skal bli sendt 3-4 måneder etter dødsfallet.
                 </BodyLong>
-                <Button
+                <PersonButtonLink
                   variant="primary"
+                  fnr={oppgave.fnr || '-'}
```

Comment:
Er det ikke bedre at vi passer på at fnr alltid er tilgjeneglig en å gi den en default verdi hvis fnr ikke eksisterer?

Reply:
Eller at vi ikke viser knappen i det hele tatt hvis fnr ikke eksisterer

Reply:
Tenker å gjøre det sånn i første omgang siden tanken er at dette skal være en midlertidig løsning. Setter knappen til disabled et par linjer ned. På denne måten er det enklere for sb å legge merke til at knappen "ikke virker" og kan si ifra dersom det er noe feil. FNR skal _teknisk sett_ ikke mangle her, men da får vi litt kjappere tilbakemelding dersom det faktisk gjør det.